test/AddingUnitTests.md

```diff
@@ -0,0 +1,65 @@
+Adding unit tests
+=================
+
+# Preface
+
+We use either:
+- [catch2](./Catch2/README.md)
+- [google tests](./googletest/README.md)
+
+To write unit tests.
```

Comment:
here and in a few places before - this sentence is just a continuation of the sentence with enumeration ,thus it shouldn't start with a capital letter

Reply:
+1 added